frontend/src/components/ChatWindow/index.js

```diff
@@ -0,0 +1,92 @@
+import { useEffect, useState, useRef } from "react";
+// import PropTypes from "prop-types";
+import "./styles.scss";
+import io from "socket.io-client";
+// import { COMMUNICATION_SVC_BASE_URL } from "../../utils/configs";
+
+import { useSelector } from "react-redux";
+import { userSelector } from "../../stores/user";
+import { matchSelector } from "../../stores/match/match.slice";
+
+const socket = io.connect("http://localhost:8005");
```

Comment:
Following the changes in the communication-service/index.js file, need to add the `path` attribute like this:
```javascript
const socket = io.connect("http://localhost:8005", {
  path: "/communication-api",
});
```
Also, why did you decide not to use the `COMMUNICATION_SVC_BASE_URL` constant?

Reply:
fixed.